src/entities/User.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  CreateDateColumn,
+  UpdateDateColumn,
+  BaseEntity,
+} from 'typeorm';
+
+import { UserEntity } from '../types';
+
+@Entity()
+export class User extends BaseEntity implements UserEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column({ length: 10 })
+  firstname: string;
+
+  @Column({
+    unique: true,
+    length: 20,
+  })
+  email: string;
+
+  @Column({
+    nullable: true,
+    length: 20,
+  })
+  password: string;
+
+  @Column({ length: 10 })
+  lastname: string;
+
+  @Column({
+    default: true,
```

Comment:
jezeli to ma byc flaga czy ktos potwierdzil konto to zmien to w ogole na status:UserStatus i tam trzymaj czy jest active,blocked,pending i tak dalej